src/components/ebay-calendar/date-utils.ts

```diff
@@ -32,7 +32,8 @@ export function getWeekdayInfo(localeName: string) {
 export function dateArgToISO(arg: DateConstructor["arguments"]) {
     if (!arg) return undefined;
     if (/^\d\d\d\d-\d\d-\d\d$/g.test(arg)) return arg;
-    return toISO(new Date(arg));
+    const date = toISO(new Date(arg));
+    return /^\d\d\d\d-\d\d-\d\d$/g.test(date) ? date : undefined;
```

Comment:
Is this always going to match all formats? What if there is only 1 day in the date?

Reply:
Yes, this is ISO format. If there is one day it will be printed with a leading zero